src/constants.js

```diff
@@ -0,0 +1,6 @@
+const CONSTANTS = {
+  availWidth: window.screen.availWidth,
```

Comment:
I can't find good data about browser support for this. Have we verified that `window.screen` is available on all browsers we support?

Reply:
I realise now this isn't used anymore anyway so I'll just remove it.